bankdroid-legacy/src/test/java/com/liato/bankdroid/utils/ExceptionUtilsTest.java

```diff
@@ -38,4 +43,45 @@ public void bankdroidifyException() throws Exception {
         // Verify that re-bankdroidifying is a no-op
         Assert.assertSame(bankdroidified, ExceptionUtils.bankdroidifyException(bankdroidified));
     }
+
+    /**
+     * Test that we can wrap exceptions without (String) constructors.
+     */
+    @Test
+    @SuppressWarnings("PMD") // This is for the stack trace printing, we really want to do it here
```

Comment:
Could we be more specific here? e.g. `@SuppressWarnings("PMD.AvoidPrintStackTrace")` ?